ui/admin/app/instance-initializers/feature-edition.js

```diff
@@ -1,6 +1,24 @@
-export function initialize(owner) {
+import fetch from 'fetch';
+
+export async function initialize(owner) {
   const service = owner.lookup('service:feature-edition');
-  service.initialize();
+  const url = '/metadata.json';
+  let edition;
+  let features;
+
+  // Attempt to load edition for the metadata JSON
+  try {
+    const response = await fetch(url);
+    const json = await response.json();
+    edition = edition || json?.license?.edition;
```

Comment:
Couldn't this have been:
```suggestion
    const edition = json?.license?.edition;
```

It doesn't look like `edition` can get initialized before this line?